src/status_im/navigation/screens.cljs

```diff
@@ -140,6 +142,10 @@
      :options   {:insets {:top? true}}
      :component communities.accounts-selection/view}
 
+    {:name      :view-channel-members-and-details
```

Comment:
consider adding new screens with :screen/<context>.<keyword>
e.g `:screen/chat.view-channel-members-and-details`

see https://github.com/status-im/status-mobile/pull/18778